docs/docs/cli.md

```diff
@@ -1,91 +1,204 @@
+                                  with videos.
+
+  --model [time_distributed|slowfast]
+                                  Model to use for inference.  [default:
+                                  time_distributed]
+
+  --checkpoint PATH               Model checkpoint path to use for
+                                  inference.     If using, model name must
+                                  also be specified.
+
+  --gpus INTEGER                  Number of GPUs to use for inference. If
+                                  not specifiied, will use all GPUs found
+                                  on machine.
+
+  --batch-size INTEGER            Batch size to use for training.
+  --save / --no-save              Whether to save out predictions to csv
+                                  file.
+
+  --dry-run / --no-dry-run        Runs one batch of inference to check for
+                                  bugs.
+
+  --config PATH                   Specify options using yaml configuration
+                                  file instead of through command line
+                                  options.
+
+  --proba-threshold FLOAT         Probability threshold for classification
+                                  between 0 and 1. If specified binary
+                                  predictions are returned with 1 being
+                                  greater than the threshold, 0 being less
+                                  than or equal to. If not specified,
+                                  probabilities between 0 and 1 are
+                                  returned.
+
+  --output-class-names / --no-output-class-names
+                                  If True, we just return a video and the
+                                  name of the most likely class. If False,
+                                  we return a probability or indicator
+                                  (depending on --proba_threshold) for
+                                  every possible class.
+
+  -y, --yes                       Skip confirmation of configuration and
+                                  proceed right to prediction.  [default:
+                                  False]
+
+  --help                          Show this message and exit.
 ```
 
-Let's go through these one by one.
+All of the above besides `file-list`, `model`, `yes`, and `checkpoint` can also be specified in a yaml file. Let's go through the flags one by one.
+
+#### `--data-dir PATH`
+
+Path to the folder containing your videos.
+
+#### `--file-list PATH`
+
+Path to a CSV file with a column for the filepath to each video you want to classify. The CSV must hae a column for `filepath`.
+
+#### `--model TEXT`
+
+There are three versions of the algorithm that ship with zamba: `time_distributed`, `slowfast`, and `european`. If `european` is passed, the model trained on European species will be run. `time_distributed` is the default. <!-- TODO: add quick description of each model><!-->
+
```

Comment:
TODO: add quick description of each model

Reply:
I think we should add an "available models" page that has the species, number of videos, kinds of sites, and more for each of the models. We can link to that page here.